Tools/DB/db.go

```diff
@@ -0,0 +1,43 @@
+package db
+
+// All configuration for connection on database
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/msosic97/korpa-backend/config"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+
+// DSN is a string that contains the necessary information to establish a connection to a specific data source or database.
+
+var DB *sql.DB
+
+//  Connection to DB
+func StartDB(config *config.Config) (*sql.DB, error) {
+	dsn := fmt.Sprintf("%s:%s@tcp(%s:%s)/%s",
```

Comment:
you can add validations for this, for example func Validate where you will check if there is DBName provided (if len = 0) or something like that